Package.swift

```diff
@@ -2,6 +2,35 @@
 
 import PackageDescription
 
+let snapshotTestingDependencies: [Target.Dependency]
```

Comment:
This file gets a whole lot messier. Suggestions on cleanup welcome!

Reply:
I just pushed this 223d567, maybe that's a lil better?